galsim/chromatic.py

```diff
@@ -162,12 +381,28 @@ def drawImage(self, bandpass, image=None, integrator=None, **kwargs):
                                 integrate.
         @param image            Optionally, the Image to draw onto.  (See GSObject.drawImage()
                                 for details.)  [default: None]
-        @param integrator       One of the image integrators from galsim.integ [default: None,
+        @param integrator       When doing the exact evaluation of the profile, this argument should
+                                be one of the image integrators from galsim.integ [default: None,
                                 which will try to select an appropriate integrator automatically.]
+                                If setupInterpolation() has been called for this object, then
+                                `integrator` should be a string, either 'midpoint' or
+                                'trapezoidal'. [default: 'trapezoidal']
```

Comment:
I think we should enable the string options for the normal integration method as well, so the user doesn't have to wade into the code of `integ.py` to see what the options are for this parameter.